app/templates/settings/settings-file.hbs

```diff
@@ -11,7 +11,10 @@
     {{/if}}
 
     <div class="settings__file-buttons">
-        <button class="settings__file-button-save-default" {{#if syncing}}disabled{{/if}}>{{res 'setFileSave'}}</button>
+        <button class="settings__file-button-save-default" {{#if syncing}}disabled{{/if}}>
+            {{#ifeq storage 'file'}}{{res 'setFileSave'}}{{/ifeq}}
+            {{#ifneq storage 'file'}}{{res 'setFileSync'}}{{/ifneq}}
```

Comment:
`setFileSync` is a noun, not a verb. Let's add a new variable for this.

Reply:
I couldn't come up with a better name than "setFileSyncVerb".